lib/navigation/consts.hpp

```diff
@@ -17,6 +17,12 @@ static constexpr core::Float kStripeDistance    = 10.0;       // m TODOLater:che
 // define sensor checks return type
 enum class SensorChecks { kUnacceptable = 0, kAcceptable };
 
+struct Quartiles {
```

Comment:
is consts.hpp the best name for this file? it seems to have picked up a bunch of definitions that aren't constants. 
(ik this is somewhat outside the scope of this PR sorry)